test/strategic-pragmatic.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { validateIri, Strategies, HTMLEntityFoundError, InvalidIriError } from '../lib/Validate';
+import { correctUris, incorrectUris } from './uris';
+import { correctUrls, incorrectUrls } from './urls';
+import { correctUrns, incorrectUrns } from './urns';
+
+describe('pragmatic strategy', () => {
+  test(`scanning for HTML entities where they exist`, () => {
+    expect(validateIri('https://example.com/a:&#xA;', Strategies.Pragmatic)).toBeInstanceOf(HTMLEntityFoundError);
+  });
+
+  test(`scanning for HTML entities where they do not exist`, () => {
+    expect(validateIri('https://example.com/lorëm-ipsum', Strategies.Pragmatic)).toBe(undefined);
+  });
+
+  test(`using implicit pragmatic strategy`, () => {
+    expect(validateIri('https://example.com/a:&#xA;')).toBeInstanceOf(HTMLEntityFoundError);
+  });
+
+  test(`an invalid iri using implicit pragmatic strategy`, () => {
+    expect(validateIri('example-of-invalid-uri')).toBeInstanceOf(InvalidIriError);
+  });
+
+  for (const correctUrl of correctUrls) {
+    test(correctUrl, () => {
```

Comment:
For these (and the following) test blocks, I suggest prefixing the test name with either "correct" or "incorrect"